src/main/java/karuna/karuna_backend/visitor/message/domain/VisitorMessageService.java

```diff
@@ -19,4 +24,18 @@ public VisitorMessageDto sendMessage(VisitorMessageCreateDto visitorMessageCreat
         emailSender.sendEmail("HARDCODED-EMAIL@EMAIL.PL", "KARUN VISTOR MESSAGE", savedVisitorMessage.getBody(), null);
         return VisitorMessageMapper.toDto(savedVisitorMessage);
     }
+
+
+    public VisitorMessageWrapper getMessages(VisitorMessageRequest visitorMessageRequest) {
+        PageRequest pageRequest = PageRequest.of(visitorMessageRequest.offset(), visitorMessageRequest.limit());
+        List<VisitorMessage> messages = visitorMessageRepository.findAllByOrderByCreatedAtDesc(pageRequest);
+        messages.forEach(message -> {
+            if (message.getBody().length() > visitorMessageRequest.bodyLenLimit()) {
+                message.setBody(message.getBody().substring(0, visitorMessageRequest.bodyLenLimit()));
+            }
+        });
+        return new VisitorMessageWrapper(messages.stream()
```

Comment:
Move it to mapper!

Reply:
I moved to mapper and right now in mapper  we can shorten message. I added new commits

Reply:
is this ok right now?

Reply:
Move createing of wrapper to mapper